iwp_website/controllers/investor_portal.py

```diff
@@ -110,6 +110,28 @@ def my_history_share(self, sortby=None, **kw):
             values
         )
 
+    @http.route(['/my/wallet/loan'], type='http', auth="user", website=True)
```

Comment:
`my` routes hurt my eyes

Reply:
I agree. It comes from odoo portal routes :  `/my/home/`, `/my/account/`. So it's there to be coherent.